src/buildHTML.js

```diff
@@ -1232,6 +1227,18 @@ groupTypes.delimsizing = function(group, options) {
             [group.value.mclass]);
 };
 
+groupTypes.verb = function(group, options) {
+    var text = buildCommon.makeVerb(group, options);
+    var body = [];
+    var i;
+    for (i = 0; i < text.length; i++) {
+        body.push(buildCommon.makeSymbol(
+            text[i], "Main-Regular", group.mode, options, ["mathtt"]));
```

Comment:
It feels a bit weird that `\verb` can handle input symbols which other commands can not, e.g. `\mathtt{γ}` doesn't work yet, but `\verb|γ|` does, with a warning to console, and `\verb|“|` doesn't even warn. Some might argue that incorrectly displaying a glyph is better than not displaying it at all, while others might worry about breaking existing content if we start supporting these symbols in the future.

So far (and [in the context of unicode input](https://github.com/Khan/KaTeX/pull/261#issuecomment-115169955)) KaTeX chose the latter approach: don't support input unless you can do so in a compatible fashion. So perhaps we should filter for those symbols we actually have Typewriter metrics for?

If we take the other approach, and deliberately allow all symbols, even those we don't have fonts or metrics for, then it might be a good idea to consider surrogate pairs in this loop here. If the input contains a surrogate pair, then we want to keep the pair intact while building a symbol node from it.